setup.py

```diff
@@ -565,7 +573,7 @@ def get_sphinx_theme_version() -> str:
     'neo4j': neo4j,
     'odbc': odbc,
     'openfaas': [],
-    'opsgenie': [],
+    'opsgenie': http,
```

Comment:
```suggestion
    'opsgenie': [],
```

Already depends on http provider